### DISCUSSION TITLE
Topics for Feb Workshop

### DISCUSSION THREAD

**Discussion by paulboyes** [COVESA/data-expert-group#6] [Q&A]
What topics would you like to cover for the first workshop?  

I feel that we should cover the Roadmap (meaning where are we going) plus 1 or 2 priority topics.  We could pull from the [roadmap page](https://wiki.covesa.global/display/WIK4/Data+Expert+Group+Roadmap) on the wiki or choose others.  

I think that complex types, relationships to other models, architectures, and security are high priority.  I also feel governance, tools and guides/tutorials/how-tos are also important and deserve a bit of coverage.

**Comment by tguild** [comment]
Best Practices
  Create outline
  Assign topic owners
  Establish basic conventions for workflow (eg issue tags from other COVESA repos) and flagging for potential How-To or standalone resources

I'm thinking BP should be a priority to avoid early adopters going too far astray and it will be a living, evolving document

**Reply by tguild** [reply]
maybe more for a Data Expert Group workshop than VSS but there is a close relation, especially at this stage
